configs/components/libffi.rb

```diff
@@ -28,12 +28,25 @@
 
   pkg.build_requires "runtime-#{settings[:runtime_project]}"
 
+  if platform.is_windows?
+    # In Windows, libffi is unable to find the C programs (like mingw) to configure and build so the path must be set first
+    c_tools_path = 'export PATH="/cygdrive/c/tools/mingw64/bin:$(PATH)"'
+  else
+    c_tools_path = ""
+  end
+
```

Comment:
I think you'll want to override the PATH above on line 24 like:

```ruby
elsif platform.is_windows?
  pkg.environment "PATH", "$(shell cygpath -u #{settings[:gcc_bindir]}):$(PATH)"
else
  ..
```
That way the `PATH` override will apply to the configure, build and install steps in this component.